medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -236,4 +239,48 @@ mod test {
             "Removed message, but shouldn't"
         );
     }
+
+    #[test]
+    fn get_blob_name_from_content_id_raises_document_not_found_in_index_error_when_not_there() {
+        let search_client = given_a_search_client_that_returns_no_results();
+        let result = when_getting_blob_name_from_content_id(search_client);
+        then_document_not_found_in_index_error_raised(result);
+    }
+
+    fn given_a_search_client_that_returns_no_results() -> impl Searchable {
+        TestAzureSearchClientWithNoResults {}
+    }
+
+    fn when_getting_blob_name_from_content_id(
+        search_client: impl Searchable,
+    ) -> Result<String, anyhow::Error> {
+        block_on(get_blob_name_from_content_id(
+            String::from("non existent content id"),
+            &search_client,
+        ))
+    }
+
+    fn then_document_not_found_in_index_error_raised(result: Result<String, anyhow::Error>) {
```

Comment:
rename to `…error_is_raised(…`, i.e. include the `is`? just noticed that g/w/t above reads perfectly up until then

Reply:
yeah agree